packages/lisk-transaction-pool/test/transaction_pool.ts

```diff
@@ -181,4 +189,20 @@ describe('transaction pool', () => {
 			);
 		});
 	});
-});
+
+	describe('#expireTransactions', () => {
+		let removeTransactionsFromQueuesStub: sinon.SinonStub;
+		let expireTransactions: () => Promise<ReadonlyArray<Transaction>>;
+
+		beforeEach(() => {
+			removeTransactionsFromQueuesStub = sandbox.stub(transactionPool as any, 'removeTransactionsFromQueues');
+			return expireTransactions = (transactionPool as any)['expireTransactions'].bind(transactionPool);
+		});
+
+		it('should call removeTransactionsFromQueues once', () => {
+			return expireTransactions().then(() =>
```

Comment:
maybe here it's good to use eventually from chai-as-promised?

Reply:
I didn't know about it, i'll check it out.